specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9538 @@
+      }
+    },
+    "/datasources/{dataSourceName}/scans/{scanName}/runs/{runId}": {
+      "get": {
+        "tags": [
+          "ScanResult"
+        ],
+        "description": "Gets the status of the scan run with ingestion details.",
+        "operationId": "ScanResult_GetScanStatus",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/dataSourceName"
+          },
+          {
+            "$ref": "#/parameters/scanName"
+          },
+          {
+            "$ref": "#/parameters/runId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/ScanResultWithIngestion"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ScanResult_GetScanStatus": {
+            "$ref": "./examples/ScanResult_GetScanStatus.json"
+          }
+        }
+      }
+    },
+    "/datasources/{dataSourceName}/scans/{scanName}:run": {
+      "post": {
+        "tags": [
+          "ScanResult"
+        ],
+        "description": "Runs the scan",
+        "operationId": "ScanResult_RunScan",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/dataSourceName"
+          },
+          {
+            "$ref": "#/parameters/scanName"
+          },
+          {
+            "$ref": "#/parameters/runIdInQuery"
+          },
+          {
+            "in": "query",
+            "name": "scanLevel",
+            "type": "string",
+            "description": "The scan level.",
+            "enum": [
+              "Full",
+              "Incremental"
+            ],
+            "x-ms-enum": {
+              "name": "ScanLevelType",
+              "values": [
+                {
+                  "value": "Full",
+                  "description": "the scan is full scan"
+                },
+                {
+                  "value": "Incremental",
+                  "description": "the scan is incremental scan"
+                }
+              ],
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "headers": {
+              "Operation-Location": {
+                "description": "Response header for long running operation.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/OperationResponse"
```

Comment:
I think the response schema for this operation should be "ScanResultWithIngestion", since this is what Get on
`/datasources/{dataSourceName}/scans/{scanName}/runs/{runId}` returns.  This is your status monitor resource.  The operation-location returned from "runScan" should be a URL to this endpoint.

Reply:
Yes, we changed the schem to "$ref": "#/definitions/ScanResultWithIngestion"

Reply:
Missed to commit on this suggestion before. Just changed the returning schema to "$ref": "#/definitions/ScanResultWithIngestion"